requirements/base.txt

```diff
@@ -11,6 +11,7 @@ whitenoise==3.2.2
 # Forms
 django-braces==1.10.0
 django-crispy-forms==1.6.1
+django_filter==1.0.4
```

Comment:
the library (and filter subclass) is also generating the form on the page (search fields for build name and plan name).

it's not strictly necessary, but it meant I didn't have to build a form. would it be worth it for me to try the other method and compare?